example/template.env

```diff
@@ -1,55 +1,68 @@
-MYSQL_IMAGE_NAME=mariadb:10.11
-
-## Mediawiki Configuration
-## Admin password
-## Passwords must be at least 10 characters.
-## Your password must be different from your username.
-## Your password must not appear within your username.
-## The password must not be in a list of very commonly used passwords. Please choose a unique password.
-MW_ADMIN_PASS=change-this-password
+# ============================================================================================
+# ESSENTIAL CONFIGURATION: Review and update entries in this section to get started
+# ============================================================================================
+#
+# MediaWiki / Wikibase Configuration
 MW_ADMIN_NAME=admin
-MW_ADMIN_EMAIL=admin@example.com
+MW_ADMIN_EMAIL=admin@mydomain.net
+# Passwords must be at least 10 characters
+# Your password must be different from your username
+# Your password must not appear within your username
+# The password must not be in a list of very commonly used passwords
+MW_ADMIN_PASS=change-this-password
+# Should be unique across all instances of this MediaWiki,
+# see https://www.mediawiki.org/wiki/Manual:$wgSecretKey
 MW_SECRET_KEY=some-secret-key
-MW_WG_ENABLE_UPLOADS=false
-
-## Jobrunner Configuration
-MAX_JOBS=1
-
-## Database Configuration
+#
+# Database Configuration
 DB_NAME=my_wiki
 DB_USER=sqluser
-DB_PASS=change-this-sqlpassword
+DB_PASS=change-this-password
+#
+# Public Hostname Configuration
+# As long as the hostnames specified here route to the IP address of this host, and the host is
+# publically acessiblethen, then the provided Reverse Proxy service (nginx-proxy) will make each
+# service publically accessible at the expected URL (e.g. http://wikibase.mydomain.net)
+WIKIBASE_PUBLIC_HOST=wikibase.mydomain.net
+WDQS_FRONTEND_PUBLIC_HOST=wdqs-frontend.mydomain.net
+QUICKSTATEMENTS_PUBLIC_HOST=quickstatements.mydomain.net
 
-## Wikibase Configuration
+# ============================================================================================
+# ADVANCED CONFIGURATION: Review and update the settings below for additional customization
+# ============================================================================================
+#
+# Mediawiki / Wikibase Configuration
+MW_WG_ENABLE_UPLOADS=false
 WIKIBASE_PINGBACK=false
-# wikibase.svc is the internal docker hostname, change this value to the public hostname
-WIKIBASE_HOST=wikibase.svc
+#
+# Jobrunner Configuration
+MAX_JOBS=1
+#
+# Service Port Configuration
+# The ports specified below are the ports each front-end service will be published to on the host.
+# These only need to be changed if they are in conflict with existing services on the host.
+# The Reverse Proxy service (nginx-proxy) will also be on the HTTP and HTTPS ports (80 and 443).
+# NOTE: Currently default ports for MariaDB/MySQL (3306), Elasticsearch (9200), and WDQS (9999) are
+# set to standard defaults and not configurable using this template. If it is ncessary to change
```

Comment:
Why make them not configurable?

Reply:
They were not before, this is a documentation update just noting that, and nlike the 3 front-end services which we have configurable ports for, these 2 have recognised "standard" defaults.  Also, we have a lot more considerations to make on the documentation bits here, but we have 3 tickets coming-up for that. The doc updates here are just a rough first pass and putting some stuff in-order on the way...